src/App.vue

```diff
@@ -28,7 +28,11 @@
       return {
         schedule: null as ScheduleEvent[] | null,
         day: null as string | null,
-        loading: true
+        loading: true,
+
+        // Configuration: mainly feature toggles about Publicis Sapient
+        // specificities and branding
+        publicisSpeedLegendEnabled: true,
```

Comment:
Here we could use Vue.js environment instead of hardcoded value: https://cli.vuejs.org/guide/mode-and-env.html